specs/data/boxTexturedUnoptimized/CesiumTexturedBoxTest.gltf

```diff
@@ -159,6 +159,14 @@
                     "mode": 4
                 }
             ]
+        },
+        "unusedMeshId": {
+            "name": "Mesh",
+            "primitives": [
```

Comment:
`primitives` is also optional so we could just make this an empty object.

See https://github.com/KhronosGroup/glTF/blob/master/specification/README.md#reference-mesh